### PR TITLE
Add ARM64 jobs in Travis-CI

### LINKED ISSUE #43: Wheel support for linux aarch64
**Summary**
Installing lazy-object-proxy on aarch64 via pip using command "pip3 install lazy-object-proxy" tries to build wheel from source code

**Problem description**
lazy-object-proxy don't have wheel for aarch64 on PyPI repository. So, while installing lazy-object-proxy via pip on aarch64, pip builds wheel for same resulting in it takes more time to install lazy-object-proxy. Making wheel available for aarch64 will benefit aarch64 users by minimizing lazy-object-proxy installation time.

**Expected Output**
Pip should be able to download lazy-object-proxy wheel from PyPI repository rather than building it from source code.

@lazy-object-proxy-team, please let me know if I can help you building wheel/uploading to PyPI repository. I am curious to make lazy-object-proxy wheel available for aarch64. It will be a great opportunity for me to work with you.

### PR DESCRIPTION
**Pull request by odidev**
Added ARM64 jobs in Travis-CI. 

Resolves #43.

### CODE REVIEW

**Comment by odidev** [comment]
@ionelmc ,

AMD64 and ARM64 `no-cov` jobs have been failed with timeout error as it is not able to get twine password and OSX python 3.7 jobs are failing with python interpreter issue, as in command `brew upgrade python`, python is being upgraded to version 3.8.4. 

Please review and let me know if any changes required.

File: .travis.yml [hunk 1]
```diff
@@ -18,75 +17,137 @@ matrix:
...
       python: 'pypy3'
-    - env:
+    - arch: amd64
+      env:
         - TOXENV=pypy3-nocov
         - TOXPYTHON=pypy3
       python: 'pypy3'
 before_install:
+  - if [[ ${TRAVIS_CPU_ARCH} != "arm64" ]]; then 
+        export LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so;
```
Comment (ionelmc) [inline .travis.yml:149]: Could this maybe be `/lib/aarch64-linux-gnu/libSegFault.so`? Is there a way to verify that (eg, you have a arm64 machine around)?

File: .travis.yml [hunk 2]
```diff
@@ -114,12 +175,18 @@ script:
...
+         python3 -m auditwheel repair $WHEEL_PATH/*.whl -w $WHEEL_PATH/
+         rm -rf $WHEEL_PATH/*linux_aarch64.whl
+      fi
       twine upload --repository-url https://test.pypi.org/legacy/ --skip-existing $WHEEL_PATH/*.whl
     fi
     )
 after_failure:
-  - more .tox/log/* | cat
-  - more .tox/*/log/* | cat
+  - travis_wait 15 more .tox/log/* | cat
```
Comment (ionelmc) [inline .travis.yml:188]: What's the point of using the waiter here?

**Comment by ionelmc** [comment]
So I've added some questions. No worries about the CI failures (there some stuff there that I need to fix for osx anyway).

File: .travis.yml (hunk 2 above)
Comment (odidev) [reply .travis.yml:191]: When the build fails in arm64, this command was giving timeout error when tested locally. After adding this code, command is executing properly.

File: .travis.yml (hunk 1 above)
Comment (odidev) [reply .travis.yml]: I have checked in my local machine and added as suggested. Please review and let me know if anymore changes required.

**Comment by odidev** [comment]
After adding https://github.com/ionelmc/python-lazy-object-proxy/commit/c69b208e864ad35086c895f377b15a0383c4f44f , travis build has been triggered in odidev travis account.

Report link: https://travis-ci.org/github/odidev/python-lazy-object-proxy/builds/709207016

I am not sure why travis build has not been triggered here.

File: .travis.yml (hunk 2 above)
Comment (ionelmc) [reply .travis.yml:191]: To be honest now I wonder why I didn't have just `cat  .tox/log/*` directly in there.

Comment (odidev) [reply .travis.yml]: I have updated that command to `cat .tox/log/*`.

**Comment by odidev** [comment]
@ionelmc ,

Thanks for adding the commits. May I know, When are you planning to make a release with aarch64 wheels to PyPI?

**Comment by ionelmc** [comment]
Seems that github didn't detect the rebase.

About releasing ... I guess today? I would like to know why the wheels are manylinux2014 instead of manylinux1. You can take a look and try the latest produced wheels here: https://test.pypi.org/project/lazy-object-proxy/1.5.1.dev5/#files

**Comment by odidev** [comment]
> Seems that github didn't detect the rebase.
> 
> About releasing ... I guess today? I would like to know why the wheels are manylinux2014 instead of manylinux1. You can take a look and try the latest produced wheels here: https://test.pypi.org/project/lazy-object-proxy/1.5.1.dev5/#files

Thanks for the reply. I am able to successfully install from the wheels available in [test-PyPI](https://test.pypi.org/project/lazy-object-proxy/1.5.1.dev5/#files). Pypa manylinux has added aarch64 support only for manylinux2014 not for manylinux1.

**Pull request closed without merging**